modules/core/shared/src/main/scala/gem/enum/GnirsReadMode.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.syntax.eq._
+import cats.instances.string._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for GNRIS Read Mode.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GnirsReadMode(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val minimumExposureTime: Int,
+  val readNoise: Int,
+  val readNoiseLow: Int
+) extends Product with Serializable
+
+object GnirsReadMode {
+
+  /** @group Constructors */ case object Bright extends GnirsReadMode("Bright", "Bright", "Bright Objects", 600, 30, 1)
+  /** @group Constructors */ case object Faint extends GnirsReadMode("Faint", "Faint", "Faint Objects", 9000, 10, 16)
+  /** @group Constructors */ case object VeryFaint extends GnirsReadMode("VeryFaint", "VeryFaint", "Very Faint Objects", 18000, 7, 32)
+  /** @group Constructors */ case object VeryBright extends GnirsReadMode("VeryBright", "VeryBright", "Very Bright/Acq./High Bkgrd.", 200, 155, 1)
+
+  /** All members of GnirsReadMode, in canonical order. */
```

Comment:
Could we put these in *logical* order instead? (e.g. Very Bright, Bright, Faint, VeryFaint)

Reply:
@jdnavarro you may need to add an artificial sort key to the database table in order to select these in the required order.